frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar_user.dart

```diff
@@ -92,10 +94,26 @@ class SidebarUser extends StatelessWidget {
 
   Widget _buildUserName(BuildContext context, MenuUserState state) {
     final String name = _userName(state.userProfile);
-    return FlowyText.medium(
-      name,
-      overflow: TextOverflow.ellipsis,
-      color: Theme.of(context).colorScheme.tertiary,
+    return GestureDetector(
+      onTap: () async {
+        final result = await OpenAIService.requestTextCompletion(
```

Comment:
I added this code in my previous PR as a sample code. You shouldn't include it here.

Remove the GestureDetector.

Reply:
sure.